pkg/metricscollector/v1beta1/common/const.go

```diff
@@ -46,6 +46,8 @@ const (
 	// Score=1.23E10
 	DefaultFilter = `([\w|-]+)\s*=\s*([+-]?\d*(\.\d+)?([Ee][+-]?\d+)?)`
 
+	TimeStampJsonKey = "timestamp"
```

Comment:
I think we should document this somewhere.

Reply:
Does this mean we need to add documentation other than the website documentation? @andreyvelich

Reply:
I meant just point about it in the website documentation.